eldritch/test_mythos.py

```diff
@@ -2264,6 +2504,129 @@ def testFastMonstersMoveAsNormal(self):
     self.assertEqual(self.monster.movement(self.state), "normal")
 
 
```

Comment:
Can I ask for a few more tests?
- mythos38 (clues to seal should decrease by 2)
- mythos57 (lose your whiskey when being arrested)
- mythos58 (health wager - test the reward and being devoured)

Reply:
Ask and ye shall receive.